src/App.js

```diff
@@ -1,19 +1,44 @@
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
+import {Component} from 'react'
 import Header from './components/Header'
 import Main from './components/Main'
 import Footer from './components/Footer'
+import SelectedBeast from './components/SelectedBeast'
 import beastObjects from './data.json'
 
 
-function App() {
-  return (
-    <>
-    <Header title='The Beastie Boys 🦏'/>
-    <Main message='Beast selection' beastObjects={beastObjects}/>
-    <Footer title='Author: Jacob Gregor'/>
-    </>
-  );
+class App extends Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      showBeast: true,
+      beast: beastObjects[0]
+    }
+  };
+    
+    modalOnClick = (beast) => {this.setState ({ 
+      showBeast: true,
+      beast: beast
+      
+    }); console.log('test')}
```

Comment:
Really nice work on this lab!

Feedback: For easier readability, indent `this.setState({` to new line and remove console.logs (only if they are no longer in use)